### ISSUE TITLE
Web Content Accessibility Guidelines (WCAG) 2.2 2023-01-30 > 2023-02-24

### ISSUE THREAD

**Issue by michael-n-cooper** [w3cping/privacy-request#112]
### Name of your specification

Web Content Accessibility Guidelines (WCAG) 2.2

### URL of your specification

https://www.w3.org/TR/2023/CRD-WCAG22-20230125/

### When does the review need to be finished?

2023-02-24

### What has changed since any previous review?

https://www.w3.org/TR/WCAG22/#substantive-changes-since-the-6-september-2022-candidate-recommendation

### Where and how should the i18n WG raise issues?

https://github.com/w3c/wcag/issues/

**Comment by npdoty** [comment]
@LJWatson is looking at this

**Comment by LJWatson** [comment]
I've reviewed the version of the WCAG 2.2 specification referenced by @michael-n-cooper and did not find anything of concern from a privacy point of view.

I'm obliged to note that the specification does not include a section on privacy (or security) considerations, as the [AG WG charter](https://www.w3.org/2019/12/ag-charter) requires. However, the few Success Criteria that have privacy or security implications do include notes that I think are a reasonable substitute and that the absence of these sections should not block WCAG 2.2 from continuing towards Recommendation.

**Comment by sandandsnow** [comment]
Thank you very much Léonie. @michael-n-cooper, we would encourage you to add the security and privacy considerations section because it will help users of the guidelines readily find and understand those considerations. The [Self-review questionnaire: security and privacy](https://w3ctag.github.io/security-questionnaire/#considerations) provides that specifications should have privacy and security considerations sections, and it is good practice for other W3C documents where such issues arise.

**Comment by pes10k** [comment]
I've filed one issue (thank you to @LJWatson for raising it), and will close this request out

- https://github.com/w3c/wcag/issues/3135
